webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -109,7 +109,7 @@ export default class AttachIssueModal extends PureComponent {
                         label={'Project'}
                         required={true}
                         onChange={this.handleProjectChange}
-                        placeholder={'Select the project containing the issue you wish to attach the message to.'}
+                        placeholder={'Select...'}
```

Comment:
Can't it just be, Select Project?

Reply:
Sure, that works for me @asaadmahmood, updated